testbeam_analysis/hit_analysis.py

```diff
@@ -292,7 +286,8 @@ def cluster_func(hits, clz, calc_cluster_dimensions):
             file_out=output_cluster_file,
             func=cluster_func,
             func_kwargs={'clz': clz,
-                         'calc_cluster_dimensions': calc_cluster_dimensions},
+                         'noisy_pixels': noisy_pixels,
```

Comment:
Not needed, but it is just a cleanup and passing all the parameters needed inside.

Reply:
Why is this a clean up? The PR started (and the subsequent troubles with the pixel_clusterizer) because of one (!) allegedly unnecessary line of code.
Let's at least not end this with 2 really unnecessary line of code ;-)

Reply:
I don’t see why function arguments are unnecessary. You can create a lot of confusion when using nonlocal scopes. And when the function has all the necessary parameters, it can be moved around freely. Not enough?

Reply:
ok